src/compile/mark/encode/tooltip.ts

```diff
@@ -88,7 +88,7 @@ export function tooltipData(
         };
 
     const title = fieldDef.title || defaultTitle(fieldDef, formatConfig);
-    const key = array(title).join(', ');
+    const key = array(title).join(', ').replaceAll(/"/g, '\\"');
```

Comment:
Huh, do we not have a utility method for this already? Does this problem not happen for other texts, axis labels or titles?

Reply:
I didn't see one, but maybe I didn't look in the right place. I'm not sure the other areas have the same problem since this one specifically generates a dictionary rather than acts as a value.